admin/src/components/ContributionMessages/ContributionMessagesFormular.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="contribution-messages-formular">
+    <div>
+      <b-form @submit.prevent="onSubmit" @reset.prevent="onReset">
+        <b-form-textarea
+          id="textarea"
+          v-model="form.text"
+          placeholder="Enter something..."
+          rows="3"
+          max-rows="6"
+        ></b-form-textarea>
+        <b-row class="mt-4 mb-6">
+          <b-col>
+            <b-button type="reset" variant="danger">{{ $t('form.reset') }}</b-button>
+          </b-col>
+          <b-col class="text-right">
+            <b-button type="submit" variant="primary">{{ $t('form.submit') }}</b-button>
+          </b-col>
+        </b-row>
+      </b-form>
+    </div>
+  </div>
+</template>
+<script>
+import { adminCreateContributionMessage } from '@/graphql/adminCreateContributionMessage'
+
+export default {
+  name: 'ContributionMessagesFormular',
+  props: {
+    contributionId: {
+      type: Number,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      form: {
+        text: '',
+      },
+    }
+  },
+  methods: {
+    onSubmit(event) {
+      this.$apollo
+        .mutate({
+          mutation: adminCreateContributionMessage,
+          variables: {
+            contributionId: this.contributionId,
+            message: this.form.text,
+          },
+        })
+        .then((result) => {
+          this.$emit('get-list-contribution-messages', this.contributionId)
+          this.$emit('update-state', this.contributionId)
+          this.form.text = ''
+          this.toastSuccess(this.$t('message.request'))
```

Comment:
I get an undefined toast message:

<img width="1646" alt="Bildschirmfoto 2022-08-30 um 11 26 53" src="https://user-images.githubusercontent.com/25344101/187401978-352eb54a-5ce9-4cf5-a3f8-9074b437ee0f.png">

Reply:
is changed